po/sv.po

```diff
@@ -0,0 +1,592 @@
+
+#: data/ui/dialogs/preferences.ui:174
+msgid "Stream public timelines"
+msgstr "Strömma offentliga tidslinjer"
+
+#: data/ui/dialogs/new_account.ui:21
+msgid "Add Account"
+msgstr "Lägg till konto"
+
+#: data/ui/dialogs/new_account.ui:25 data/ui/views/base.ui:26
+#: src/Dialogs/NewAccount.vala:85
+msgid "Back"
+msgstr "Tillbaka"
+
+#: data/ui/dialogs/new_account.ui:48 src/Dialogs/NewAccount.vala:84
+msgid "Next"
+msgstr "Nästa"
+
+#: data/ui/dialogs/new_account.ui:129
+msgid "What is your instance?"
+msgstr "Vilken är din instans?"
+
+#: data/ui/dialogs/new_account.ui:191
+msgid "<a href=\"https://joinmastodon.org/\">Don't have one yet?</a>"
+msgstr "<a href=\"https://joinmastodon.org/\">Har du ännu ingen?</a>"
+
+#: data/ui/dialogs/new_account.ui:242
+msgid "Enter authorization code"
+msgstr "Ange auktoriseringskod"
+
+#: data/ui/dialogs/new_account.ui:276
+msgid "<a href=\"tootle://manual_auth\">Try manual authorization</a>"
+msgstr "<a href=\"tootle://manual_auth\">Försök med manuell auktorisering</a>"
+
+#: data/ui/widgets/accounts_button.ui:64 src/Views/Conversations.vala:6
+msgid "Conversations"
+msgstr "Konversationer"
+
+#: data/ui/widgets/accounts_button.ui:93 src/Views/Favorites.vala:6
+msgid "Favorites"
+msgstr "Favoriter"
+
+#: data/ui/widgets/accounts_button.ui:122 src/Views/Bookmarks.vala:6
+msgid "Bookmarks"
+msgstr "Bokmärken"
+
+#: data/ui/widgets/accounts_button.ui:152 src/Views/Lists.vala:59
+msgid "Lists"
+msgstr "Listor"
+
+#: data/ui/widgets/accounts_button.ui:204
+msgid "POSTS"
+msgstr "INLÄGG"
+
+#: data/ui/widgets/accounts_button.ui:247
+msgid "FOLLOWING"
+msgstr "FÖLJER"
+
+#: data/ui/widgets/accounts_button.ui:290
+msgid "FOLLOWERS"
+msgstr "FÖLJARE"
+
+#: data/ui/widgets/accounts_button.ui:344
+msgid "Refresh"
+msgstr "Uppdatera"
+
+#: data/ui/widgets/accounts_button.ui:357
+msgid "Preferences"
+msgstr "Inställningar"
+
+#: data/ui/widgets/accounts_button.ui:371
+msgid "About"
+msgstr "Om"
+
+#: data/ui/widgets/accounts_button.ui:397 src/Views/Search.vala:43
+msgid "Accounts"
+msgstr "Konton"
+
+#: data/ui/widgets/accounts_button_item.ui:60
+msgid "Forget"
+msgstr "Glöm"
+
+#: data/ui/widgets/accounts_button_item.ui:88
+msgid "Title"
+msgstr "Titel"
+
+#: data/ui/widgets/compose_attachment.ui:29
+msgid "Describe for the visually impaired"
+msgstr "Beskriv för synskadade"
+
+#: data/ui/widgets/compose_attachment.ui:90
+msgid "Unknown"
+msgstr "Okänd"
+
+#: data/ui/widgets/compose_attachment.ui:113 data/ui/widgets/list_item.ui:74
+#: src/Widgets/Status.vala:243
+msgid "Delete"
+msgstr "Ta bort"
+
+#: data/ui/widgets/list_item.ui:26 src/Dialogs/ListEditor.vala:87
+msgid "Untitled"
+msgstr "Namnlös"
+
+#: data/ui/widgets/list_item.ui:42
+msgid "Edit"
+msgstr "Redigera"
+
+#. Please do not remove the credits below. You may add your own, but keep the existing ones intact.
+#. TRANSLATORS: Replace this with your name. It will be displayed in the About dialog.
+#: src/Build.vala:15
+msgid " "
+msgstr " "
```

Comment:
Please fill in this blank so that you can appear as a locale maintainer. This will be used in the About dialog.